lib/train/platforms/detect/uuid.rb

```diff
@@ -22,12 +22,12 @@ def find_or_create_uuid
       elsif @platform.windows?
         windows_uuid
       else
-        if @platform[:uuid_command]
-          result = @backend.run_command(@platform[:uuid_command])
+        if @platform.uuid_command
+          result = @backend.run_command(@platform.uuid_command)
           return uuid_from_string(result.stdout.chomp) if result.exit_status.zero? && !result.stdout.empty?
         end
 
-        raise 'Could not find platform uuid! Please set a uuid_command for your platform.'
+        raise Train::TransportError, 'Could not find platform uuid! Please set a uuid_command for your platform.'
```

Comment:
I think we should remove this raise all together...but I'll put it up for discussion.

Currently this shows as a warning on an InSpec run.

Reply:
3.0 removes any uuid pulling so it will not anymore